src/utils.js

```diff
@@ -70,11 +70,13 @@ function stringifyWebhooksToBase64(webhooks) {
 let gzipPromise;
 if (isNode()) gzipPromise = util.promisify(zlib.gzip);
 /**
- * @param {*} value
+ * Gzip provided value, otherwise returns null.
+ * @param value
+ * @return {Promise<null|Buffer>}
  */
 async function maybeGzipValue(value) {
-    if (!isNode()) return value;
-    if (value == null) return value;
+    if (!isNode()) return null;
+    if (value == null) return null;
```

Comment:
Please use `return` or `return undefined`. I don't think we return `null` anywhere in the client.